test/signer_mock.go

```diff
@@ -0,0 +1,58 @@
+package test
```

Comment:
Not a blocker, but we could make this an internal package to signal to users of the project that it shouldn't be used externally (an internal package will actually prevent it from being imported in another project). Not sure if that's what we want here though.